README.md

```diff
@@ -62,9 +62,12 @@ Every project runs differently. Follow instructions in the main `README.md` docu
 
 5. Create a branch and start to develop.
 
-6. Test your changes.
+Do not forget about creating unit and acceptance tests if needed. For the unit tests, follow the instructions specified in the main `README.md` document of the given project. For the acceptance tests, go to the corresponding directory inside `tests` directory for details.
+Information how to run changes on cluster without Docker Image can be found in [Develop a service locally without using Docker](docs/kyma/docs/035-gs-local-develop-no-docker.md)
+
+> For more details about testing go to [Testing Kyma](docs/kyma/docs/026-details-testing.md)
```

Comment:
``>**NOTE:** For more details about testing, go to the [Testing Kyma](docs/kyma/docs/026-details-testing.md) document.``